.ci/run.sh

```diff
@@ -9,10 +9,27 @@ set -e
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
+install_docker() {
+	docker_version=${docker_version:-19.03}
+	pkg_name="docker-ce"
+	repo_url="https://download.docker.com/linux/ubuntu"
+	curl -fsSL "${repo_url}/gpg" | sudo apt-key add -
+	sudo -E add-apt-repository "deb [arch=${arch}] ${repo_url} $(lsb_release -cs) stable"
+	sudo -E apt-get update
+	docker_version_full=$(apt-cache madison $pkg_name | grep "$docker_version" | awk '{print $3}' | head -1)
+	sudo -E apt-get -y install "${pkg_name}=${docker_version_full}"
+}
+
 pushd "${tests_repo_dir}"
 .ci/run.sh
 testcidir=$(dirname "$0")
 
+if [ $arch == "arm64" ]; then
+	command -v docker >/dev/null 2>&1 && "${testcidir}/../cmd/container-manager/manage_ctr_mgr.sh" docker remove
+	echo "reinstall docker 19.03 for arm64"
+	install_docker 19.03
```

Comment:
this `install_docker` is distro specific, I think you should rename it to `install_docker_ubuntu` or something else that contains the `ubuntu` word

Reply:
ok. thanks